charts/app-config/values-integration.yaml

```diff
@@ -23,9 +23,14 @@ govukApplications:
       enabled: true
       annotations:
         alb.ingress.kubernetes.io/load-balancer-name: assets-origin
+        alb.ingress.kubernetes.io/group.name: assets-origin
+        alb.ingress.kubernetes.io/group.order: '1'
```

Comment:
Might be worth making these `10`, `20`, `30` initially so that we have the option of making future changes without necessarily having to do lots of renumbering. (insert BASIC joke here)

The range is 1-1000 so we have plenty of headroom.